internal/testutil/treedump.go

```diff
@@ -74,7 +74,12 @@ func TreeDumpEntry(entry *fsutil.Entry) string {
 		return fmt.Sprintf("dir %#o", fperm)
 	case fs.ModeSymlink:
 		return fmt.Sprintf("symlink %s", entry.Link)
-	case 0: // Regular
+	case 0:
+		// Hard link
```

Comment:
```suggestion
		// Hard link.
```

Very nitpicky but we prefer punctuation in comments :)

Reply:
Fixed in [4090b4c](https://github.com/letFunny/chisel/pull/17/commits/4090b4c4c27eda1bd386bb1757572a4529887ad9). However, it seems this change should be reverted since we don't need this in `TreeDumpEntry`.